docs/index.rst

```diff
@@ -29,3 +29,10 @@ Welcome to the documentation of the ohsome API!
    group-by
    time
    filter
+   
+Additional Links:
+-----------------
+
+* `API Code development <https://github.com/GIScience/ohsome-api>`_
+* `Swagger API playground <https://api.ohsome.org/v1/swagger-ui.html>`_
```

Comment:
Would it be better to have a `/stable` link or is it sufficient to have `/v1` since we have to change a lot of things here for a new (future) version?

Reply:
As far as I can tell, there is no /stable link yet :)

And, if we have them, I guess we could link both: the /stable instance, as well as the /development

Reply:
> As far as I can tell, there is no /stable link yet :)
No there isn't. But we can create one if that helps ;)

> And, if we have them, I guess we could link both: the /stable instance, as well as the /development
If you want to publish that (unstable) development instance, yes :thinking:

Reply:
Alright :) I agree, that pointing to /stable instead of a specific version would be smarter as we won't have to adapt it everytime for a new version.

So should the development instance just be for internal usage then? Thought we could also make it externally accessible, e.g. if someone needs a "quick & dirty" feature for some measurement or such. Then just linking it there (without providing a specific docu) could be useful.

Reply:
```suggestion
* `Swagger API playground <https://api.ohsome.org/stable/swagger-ui.html>`_
```

Reply:
Alright :)
No other change then needed I think.. will commit that suggestion and then merge as soon as the link is working.